plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -73,6 +73,8 @@ public class IcebergConfig
     private double minimumAssignedSplitWeight = 0.05;
     private Optional<String> materializedViewsStorageSchema = Optional.empty();
     private boolean sortedWritingEnabled = true;
+    private int maxSplitsPerSecond = Integer.MAX_VALUE;
+    private int maxOutstandingSplits = 1000;
```

Comment:
document how this value was picked

Reply:
I picked this one to be consistent with Hive, Hudi and Delta, which use this same default value for `maxOutstandingSplits`

Reply:
```
// Default value picked after Hive, Hudi and Delta
```

Reply:
Comment added